Question: https://datalemur.com/questions/coin-fairness-test

**Solution**:

**Null Hypothesis**: Coin is a Fair Coin

**Alternate Hypothesis**: Coin is a not Fair Coin biased towards the tail


As we can see in the question, **out of 10 times** only **1 time head** is observed. To calculate we probability of this event we can use Binomial Distribution Formula.

$$
Binomial \ probability:\ ^{n}C_{r} \  p^{r} \ (1-p)^{n-r}
$$
$$
n = 10
$$
$$
r = 1
$$
$$
p = 0.5
$$


In [14]:
import math
#Calculating binomial probability
n = 10
r = 1
p = 0.5
(math.factorial(n) / ((math.factorial(r))*math.factorial(n-r))) * (p**r) * ((1-p)**(n-r))

0.009765625

Now, in this probability of the given case is the p-value

$$
p-value \ = 0.009765625 < 0.05
$$

**p-value is less than 0.05, So we can reject the null hypthesis**